database/migrations/1550596476_create_users_table.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+CREATE TABLE users
+(
+  id       UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  username VARCHAR(100) NOT NULL,
+  password TEXT         NOT NULL,
+  token    UUID             DEFAULT gen_random_uuid()
```

Comment:
@ant31 wanted the token to be on the tenant level, as these are likely going to be used for machine authentication and shouldn't stop working when a user is deleted.

Reply:
Makes sense. Seems like that info didn't reach me.

Reply:
Can we do this in a follow up PR. As this changes quite some things, including the authentication which then needs to happen on a tenant basis.

Reply:
Sounds good to me